daemon/k8s_watcher.go

```diff
@@ -1449,6 +1449,30 @@ func getUpdatedCNPFromStore(ciliumV2Store cache.Store, cnp *cilium_v2.CiliumNetw
 	return serverRule, nil
 }
 
+func (d *Daemon) recalculatePolicyFromCNPUpdate(ciliumV2Store cache.Store, cnp *cilium_v2.CiliumNetworkPolicy) {
+	scopedLog := log.WithFields(logrus.Fields{
+		logfields.CiliumNetworkPolicyName: cnp.ObjectMeta.Name,
+		logfields.K8sAPIVersion:           cnp.TypeMeta.APIVersion,
+		logfields.K8sNamespace:            cnp.ObjectMeta.Namespace,
+	})
+
+	scopedLog.Infof("recalculating policy from CNP update")
```

Comment:
This info message is a bit misleading as it is only printed if old and new spec are equal and annotations have changed.

Reply:
Fixed.